debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -591,12 +591,12 @@ alterView
 
 alterSpecification
     : tableOption (','? tableOption)*                               #alterByTableOption
-    | ADD COLUMN? uid columnDefinition (FIRST | AFTER uid)?         #alterByAddColumn
+    | ADD COLUMN? ifNotExists? uid columnDefinition (FIRST | AFTER uid)?         #alterByAddColumn
```

Comment:
Could you add a comment that `ifNotExists` is specific to MariaDB? We haven't consistently marked those extensions so far, but I think we should begin doing so.